src/Server.cpp

```diff
@@ -178,24 +178,21 @@ void Server<IoType, SwitchActivityInfo, SwitchCalcGaps>::doLoop()
 				log_msg("Error: %s() returned without fd ready", handler2str(g_pApp->m_const_params.fd_handler_type));
 			continue;
 		}
-
 		// handle arrival and response
 		int accept_fd = (int)INVALID_SOCKET; // TODO: use SOCKET all over the way and avoid this cast
 		bool do_update = false;
 		for (int ifd = m_ioHandler.get_look_start(); (numReady) && (ifd < m_ioHandler.get_look_end()); ifd++) {
 			actual_fd = m_ioHandler.analyzeArrival(ifd);
-
 			if (actual_fd){
-				assert( g_fds_array[actual_fd] &&
+				assert(g_fds_array[actual_fd] &&
 						"invalid fd");
 
 				if (!g_fds_array[actual_fd])
 				{
 					print_log("fd received was larger than expected, ignored.", actual_fd);
 					/* do nothing invalid fd*/
 				}
-				else
-				{
+				else{
```

Comment:
in case you want to commit white space fixes, then the best would be to do distinct commit that only fixes the white spaces without messing it with a code change commit that requires more attention